File: Modul1/C/PRAK110-2310817210003-Muhammad_Azrianzan.c
#include <stdio.h>
#include <math.h>

int main() {
    int alas = 5;
    int tinggi = 12;
    printf("Diketahui :\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);
    printf("\n");
    printf("Jawab :\n");

    int sisiB = sqrt(pow(alas, 2) + pow(tinggi, 2));

    printf("Sisi A = %d cm\n", tinggi);
    printf("Sisi B = %d cm\n", sisiB);
    printf("Sisi C = %d cm\n", alas);
    int keliling = alas + tinggi + sisiB;
    printf("Keliling = %d cm\n", keliling);
    int luas = 0.5 * alas * tinggi;
    printf("Luas = %d cm", luas);
    return 0;
}

File: Modul1/Python/PRAK110-2310817210003-Muhammad_Azrianzan.py
import math

alas = 5
tinggi = 12
print("Diketahui :")
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")
print("")
print("Jawab :")

sisiB = math.sqrt(alas**2 + tinggi**2)

print(f"Sisi A = {tinggi} cm")
print(f"Sisi B = {sisiB:.0f} cm")
print(f"Sisi C = {alas} cm")
keliling = alas + tinggi + sisiB
print(f"Keliling = {keliling:.0f} cm")
luas = 0.5 * alas * tinggi
print(f"Luas = {luas:.0f} cm")
